test/mind_the_gapp_web/controllers/arrival_time_controller_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule MindTheGappWeb.ArrivalTimeControllerTest do
+  use MindTheGappWeb.ConnCase, async: true
+  use Timex
+
+  setup_all :create_fake_arrival_time_data
+
+  def create_fake_arrival_time_data(_context) do
+    clear_database()
```

Comment:
Same comment here. Tests should be running in transactions so no need to manually clean up the data.